js/renovation/viz/core/tooltip.tsx

```diff
@@ -266,6 +282,21 @@ export class Tooltip extends JSXComponent(TooltipProps) {
     return getFormatValue(value, specialFormat, { format, argumentFormat });
   }
 
+  @Method()
+  isEnabled(): boolean {
+    return this.props.enabled;
+  }
+
+  @Method()
+  isShared(): boolean {
+    return !!this.props.shared;
```

Comment:
Is not `shared` a boolean type? Why do we use a `!!` operator?

Reply:
yes, it is copy paste from the original code :)